README.md

```diff
@@ -3,7 +3,7 @@
 <img src="docs/images/image-banner.png" align="middle" width="3000"/>
 
 # Unity ML-Agents Toolkit (Beta)
-[![docs badge](https://img.shields.io/badge/docs-reference-blue.svg)](https://github.com/Unity-Technologies/ml-agents/tree/latest_release/docs/)
+[![docs badge](https://img.shields.io/badge/docs-reference-blue.svg)](https://github.com/Unity-Technologies/ml-agents/tree/0.15.0/docs/)
```

Comment:
why this change? I thought we always wanted it to point to `latest_release`.

Reply:
when a user goes on the branch with the tag 0.15, the docs should point towards 0.15.
I think we only want to point to latest release on master but on a release branch, we want to point to the release branch.

Reply:
This will cause the markdown link check to fail (although only the nightly job checks remote links) since the tag doesn't exist.